applications/nrf5340_audio/src/audio/Kconfig.defaults

```diff
@@ -19,56 +19,3 @@ config CMSIS_DSP
 # Required for calculating test tones
 config CMSIS_DSP_FASTMATH
 	default y
-
```

Comment:
Creep2: (Comment in wrong place): The two test-tone dependent Kconfigs should depend on if test-tone is compiled in or not.